R/derive_param_lasta.R

```diff
@@ -0,0 +1,358 @@
+#%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%|
+#' @description
+#'     Derived Last Disease Assessment
```

Comment:
add "up to first Progressive Disease"

Reply:
source_pd is optional, so this is in the details section.

I can change to :  Derived Last Disease Assessment (up to first Progressive Disease if source_pd is set as an argument)

But feel it better as is.

Happy to be overruled.

Reply:
ok use "(optionally) up to first Progressive Disease". we've been adding similar in all functions to explain how we've put this onco-specific piece in vs some more generic versions of similar functions in admiral core

Reply:
will do, thanks Ross.